torch/csrc/functionalization/Module.cpp

```diff
@@ -0,0 +1,33 @@
+#include <torch/csrc/functionalization/Module.h>
+
+#include <ATen/FunctionalizeFallbackKernel.h>
+
+namespace torch::functionalization {
+
+void initModule(PyObject* module) {
+  auto m = py::handle(module).cast<py::module>();
+
+  // Create a `torch._C._functionalization` Python module.
```

Comment:
no action needed: if we're going to make a dedicated `_functionalization` module, we might want to consider moving the existing pybind-ed C helper functions here too (all of the `_functionalize_*` methods in https://github.com/pytorch/pytorch/blob/main/torch/csrc/autograd/python_torch_functions_manual.cpp#L606)